checks.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+res_pip_list=$(pip freeze)
+if [[ ! $(echo $res_pip_list | grep black) ]] ; then
+  pip install black
+fi
+if [[ ! $(echo $res_pip_list | grep flake8) ]] ; then
+  pip install flake8
+fi
+if [[ ! $(echo $res_pip_list | grep isort) ]] ; then
+  pip install isort
+fi
+if [[ ! $(echo $res_pip_list | grep mypy) ]] ; then
+  pip install mypy
+fi
+
+update=0
+while getopts "u" OPT
+do
+  case $OPT in
+    u) update=1
+       ;;
+    *) ;;
+  esac
+done
+
+res_black=$(black . --check 2>&1)
+if [[ $(echo $res_black | grep reformatted) ]] ; then
+  if [ $update == 1 ] ; then
+    echo "Failed with black. The code will be formatted by black."
+    black .
+  else
+    echo "Failed with black."
+    echo "$res_black"
+    exit 1
+  fi
+else
+  echo "Success in black."
+fi
+
+res_flake8=$(flake8 .)
+if [[ $res_flake8 ]] ; then
+  echo "Failed with flake8."
+  echo "$res_flake8"
```

Comment:
The error messages are sometimes very long and users may not be able to see `Failed with flake8`. How about flipping these two lines?
```suggestion
  echo "$res_flake8"
  echo "Failed with flake8."
```